modules/ROOT/pages/design-create-publish-api-specs.adoc

```diff
@@ -41,13 +45,17 @@ You can publish an API specification to Anypoint Exchange for use in API Manager
 
 == How API Designer Relates to Other MuleSoft Products
```

Comment:
This is a robust title. 
Try to make all titles at the same hierarchical level parallel.